[BRO-48] Enable ZK proof generations with new company identities (#392)
Concordium/concordium-node-sdk-js: packages/sdk/CHANGELOG.md

### Added

- Add `legalCountry` as an allowed attribute for set/not-set membership proofs.

> - `IDENTITY_SUBJECT_SCHEMA` implemented missed attributes `lei`, `legalName`, `legalCountry`, `businessNumber`, `registrationAuth`

Reviewer: ```suggestion
- Add attributes `lei`, `legalName`, `legalCountry`, `businessNumber`, and `registrationAuth` to `IDENTITY_SUBJECT_SCHEMA`.
```
